src/Client.js

```diff
@@ -172,7 +172,7 @@ class Client extends EventEmitter {
    * @typedef {Object} JobListOptions
    * @property {Object} query - Filter jobs by specifying one or more comparison operators per property.
    *   Comparison operators must be prefixed with '$' and only the following are supported: $eq, $ne,
-   *  $gt, $gte, $lt, $lte.  Queries may contain multiple properties and each property may
+   *  $gt, $gte, $lt, $lte, $or, $nor, $and,$not, $type, $exists .Queries may contain multiple properties and each property may
    *  contain multiple comparison operators.
```

Comment:
I think we might need to add separate paragraphs to group the operators as we now have comparison, logical and element operators, $or, $nor, $and,$not are logical, $type, $exists are element operators